app/public/dist/client/locales/en/translation.json

```diff
@@ -1256,383 +1256,383 @@
+    "SOAK": "Deal [20,40,80,SP] SPECIAL to the target and restore 10 PP to all allied Pokémon.",
+    "ORIGIN_PULSE": "A wave travels horizontally across the battlefield, dealing [100,SP] SPECIAL to all enemy Pokémon in its path.",
+    "SEED_FLARE": "Emit a shockwave, deal [30,SP] SPECIAL to all enemy Pokémon in a 5 tile radius and decrease their SPE_DEF by 2.",
+    "IRON_TAIL": "Deal [20,40,80,SP] SPECIAL to the target and increase the user's DEF by [1,3,5].",
+    "BLAST_BURN": "Deal [30,60,120,SP] SPECIAL in an area around the user.",
+    "CHARGE": "Increase the ATK and ATK_SPEED of all allied Pokémon by [20,SP]%.",
+    "DISCHARGE": "Deal [25,50,75,SP] SPECIAL to nearby enemy Pokémon and inflict them with PARALYSIS for 5 seconds.",
+    "BITE": "Deal [40,80,120,SP] SPECIAL to the target and heal the user for 30% of the damage dealt. Cause the target to FLINCH for 5 seconds.",
+    "AQUA_TAIL": "Deal [30,60,100,SP] SPECIAL to the target and gain [30,60,100,SP] SHIELD.",
+    "DRAGON_TAIL": "Deal [30,60,100,SP] SPECIAL to the target and increase the user's DEF and SPE_DEF by [1,2,3,SP].",
+    "DRAGON_BREATH": "Deal [25,50,100,SP] TRUE to all enemy Pokémon behind the target.",
+    "ICICLE_CRASH": "Deal [20,40,80,SP] SPECIAL in an area around the target.",
+    "ROOT": "Deal [10,20,40,SP] SPECIAL to all adjacent enemy Pokémon and make them LOCKED for 4 seconds. Heal the user and adjacent allied Pokémon for [10,20,40,SP] HP.",
+    "TORMENT": "Increase the user's ATK_SPEED by [20,40,60,SP]%.",
+    "STOMP": "Deal SPECIAL equal to [3,4,5,SP] x ATK to the target.",
+    "PAYBACK": "Deal [15,30,60,SP] SPECIAL to the target, doubled if the user has less than 50% HP.",
+    "SHADOW_BALL": "Deal [40,60,100,SP] SPECIAL to the target and decrease the SPE_DEF of all enemy Pokémon by 1.",
+    "BUG_BUZZ": "Deal [20,40,80,SP] SPECIAL to the target, doubled if the target suffers from PARALYSIS.",
+    "STRING_SHOT": "Deal [10,20,50,SP] SPECIAL to the target and inflict PARALYSIS for 5 seconds.",
+    "VENOSHOCK": "Deal [30,60,120,SP] SPECIAL to the target, doubled if the target suffers from POISONNED.",
+    "LEECH_LIFE": "Deal [20,40,80,SP] SPECIAL to the target and heal the user for the same amount of HP.",
+    "HAPPY_HOUR": "Increase the ATK of all allied Pokémon by [2,5,8,SP].",
+    "TELEPORT": "Teleport the user to a corner of the battlefield. The user's next attack deals [15,30,60,SP] extra SPECIAL.",
+    "NASTY_PLOT": "Increase the user's ATK by [10,SP].",
+    "TAKE_HEART": "The user harnesses bravery to cure their status conditions and increase their ATK by [8,SP] and SPE_DEF by [4,SP].",
+    "THIEF": "Steal as many of the target's items as the user can carry and deal [15,30,60,SP] SPECIAL.",
+    "STUN_SPORE": "Deal [15,30,60,SP] SPECIAL to the target and all enemy Pokémon adjacent to it and inflict PARALYSIS for 5 seconds.",
+    "METEOR_MASH": "Deal 3 x [15,30,60,SP] SPECIAL to the target and increase the user's ATK by [2,SP].",
```

Comment:
In this scenario we explicitely say "3 times" to insist on the fact that these are separate attacks, for the same reason why we explicitely say "double damage"

Reply:
Ah that was unclear in the original, thanks for the clarification. Fixed.